data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,1025 @@
+	to offer
+		has "<uhai> 12: Outfitter Upgrade: done"
+		not "language: Wanderer"
+		not "First Contact: Wanderer: offered"
+		not "First Contact: Wanderer (Before Hai): offered"
+		not "unfettered: outcast"
+	to fail
+		has "<uhai> 18: Hai Language: failed"
+	on enter "Ka'ch'chrai"
+		dialog `Judging by the population compared to the other worlds, this appears to be the capital of these aliens. You should land to introduce yourself as a diplomat between them and the Hai.`
+
+mission "<uhai> 18: Hai Language"
+	minor
+	name `Meet and understand the Wanderers`
+	description `You need to be able to act as a diplomat between the Hai and the aliens up north. First go and meet those aliens. Then go to <stopovers> to see if He-Yi will provide a translator for you, or get one through other means.`
+	source "Darkcloak"
+	stopover "Warfeed"
+	to offer
+		has "<uhai> 12: Outfitter Upgrade: done"
+		not "language: Wanderer"
+	on offer
+		conversation
+			`Moving through the spaceport, you notice some buildings do not seem as crumbling as before. You wonder if the Unfettered are the ones who invested into repairs, or if the new Hai arrivals brought this about. Whatever the case is, this does not look like a place of conflict anymore. It looks like a place of hope.`
+			`	An Unfettered Herald - recognizable with their peculiar bird mask - approaches and notices you looking around. "Incredible, is it not? To see hope on the face of buildings and on the face of people. I am here to say that it's time you learn how to communicate with us - with the True language of the Hai. You also need to be able to make demands to the birds, and there is only one appropriate language for it. We have learned that they too speak Hai, even if they disgrace our language by doing so."`
+			choice
+				`	"How can I learn Hai?"`
+				`	"Is there some kind of translator?"`
+					goto translator
+			`	"Well, to be frank, we don't have the ability to teach all of its nuances to you in time. However, we believe that our brethren do have some way of doing that, given how apt they are with languages and how keen they are to meet new races."`
+			label translator
+			`	"We think our Hai brethren may indeed have what we need, but we need to find a way to obtain it.`
+			`	"They are not ready to meet the birds yet.`
+				to display
+					not "event: hai-human resolution announced"
+			`	"Perhaps you can go to them to get a translator, although that is not the preferred way. But maybe they are ready to meet the birds now.`
+				to display
+					has "event: hai-human resolution announced"
+			`	"In the meantime, you should go and meet the aliens up North and speak with them, representing us peacefully. They may not allow you to land on most of their planets. They may have grown distrusful of newcomers - probably because of our own actions. Go and see how pathethic these birds are, and then you should go to He-Yi about that translator."`
```

Comment:
```suggestion
			`	"In the meantime, you should go and meet the aliens up North and speak with them, representing us peacefully. They may not allow you to land on most of their planets. They may have grown distrustful of newcomers - probably because of our own actions. Go and see how pathetic these birds are, and then you should go to He-Yi about that translator."`
```